docs/DeveloperGuide.adoc

```diff
@@ -809,6 +811,69 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 |`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
 
 |`*` |user with many persons in the address book |sort persons by name |locate a person easily
+
+|`**` |user with many persons having long names in the address book |auto-fill the name as I key in the command | key in a person’s name without spelling out all names completely
```

Comment:
Is it necessary to separate the '*'s with a space in between?
Like the ones above.

Reply:
And examples seem to follow a desc order of importance of use stories.

Reply:
Like you have mentioned we should standardize either all have space or all dont have space. They didnt explicitly mention this but I think descending order better.